### ISSUE TITLE
Feature Request for MultiZarrToZarr to test whether input chunk sizes are identical

### ISSUE THREAD

**Issue by arschultz** [fsspec/kerchunk#136]
I have input zarr datasets which feature identical chunk sizes except for the unlimited dimension, which is time. Each of my input datasets has trailing chunks which differ from one another. I found that MultiZarrToZarr was able to create a new output json from these inputs. The time chunks of varying size then made the resulting combined dataset unable to be loaded in full by xarray, which raised a long, hard-to-parse error traceback.

I was surprised to find that despite being unable to be load the full dataset or a full time series into Xarray, the resulting zarr dataset attributes could be queried by Xarray without issue. Individual time steps could also be loaded, as could time series which were only drawn from one of the input datasets.

I understand that there are efforts being investigated to support unequal chunk sizes, but for now it might be helpful to add in a test to MultiZarrToZarr to inform the user that input datasets need to have identical chunks (well, apart from the last one).

**Comment by martindurant** [comment]
Agree, this test would be reasonable. I thought I added it, but apparently not. We would already be able to do the check at the end of the first pass, I think.

**Comment by arschultz** [comment]
Ah, unfortunately no. I just wanted to identify this potential enhancement to MultiZarrToZarr.
